crates/swift-bridge-cli/src/clap_exec.rs

```diff
@@ -34,3 +37,21 @@ fn handle_create_package(matches: &ArgMatches) {
 
     create_package(config);
 }
+
+/// Executes the `parse-bridges` command
+fn handle_parse_bridges(matches: &ArgMatches) {
+    const DEFAULT_OUTPUT_DIR: &str = "generated";
+
+    let crate_name = matches
+        .get_one::<String>("crate-name")
+        .unwrap(); // required
+    let source_files: Vec<&String> = matches.get_many("source-files")
+        .unwrap()
+        .collect(); // required
+    let output = matches.get_one::<String>("output")
+        .map(Path::new)
+        .unwrap_or_else(|| Path::new(DEFAULT_OUTPUT_DIR));
```

Comment:
Let's remove the default and require an explicit output directory.

Then we can consider introducing a default in the future if we decide that needing to specify the directory yourself is too cumbersome.

Reasons:

1. One fewer thing to figure out during this review. Adding a default would be a non-breaking change.

2. Fewer defaults helps teach the user about how things work / what is configurable

3. It is likely that users will write this command once and then re-use it, so it's okay if it's a little longer

Reply:
Good point, done.

I've force-pushed the changes.